app/content-panel/blogs/[id]/page.tsx

```diff
@@ -0,0 +1,163 @@
+'use client';
+
+import React, { useState, useEffect } from 'react';
+import styles from './BlogDetailsPage.module.scss';
+import BlogDetailsCard from '../../../../components/reusable-components/blogDetails-card/BlogDetailsCard';
+import useGetBlogDetails from '../../../../api/queries/blogs/getBlogDetails';
+
+interface Author {
+  first_name: string;
+  last_name: string;
+}
+
+interface BlogDetailsData {
+  title: string;
+  image: string;
+  content: string;
+  author: Author;
+  publishDate: string;
+  tags: string[];
+}
```

Comment:
Try to use consistent naming in the interfaces. Here you are using snake case in Author( first_name) and camel case in BlogDetailsData(publishDate).
Check other components as well so we have consistent naming in the project.

Reply:
Lets use camelCase for naming the properties inside the interface in this component.
I will make sure to go though the project and fix this inconsistencies.

Reply:
ok